ksql-engine/src/test/resources/query-validation-tests/joins.json

```diff
@@ -10,259 +10,258 @@
   "tests": [
     {
       "name": "stream stream left join",
-      "format": ["AVRO", "JSON"],
       "statements": [
-        "CREATE STREAM TEST (ID bigint, NAME varchar, VALUE bigint) WITH (kafka_topic='left_topic', value_format='{FORMAT}', key='ID');",
-        "CREATE STREAM TEST_STREAM (ID bigint, F1 varchar, F2 bigint) WITH (kafka_topic='right_topic', value_format='{FORMAT}', key='ID');",
+        "CREATE STREAM TEST (ID bigint, NAME varchar, VALUE bigint) WITH (kafka_topic='left_topic', value_format='DELIMITED', key='ID');",
```

Comment:
not sure we want to change this. The `{FORMAT}` was recently introduced so we can verify against different formats. @rodesai may have more to say

Reply:
Yes. I think this is working off an older commit before the @rodesai's updates to the `QueryValidationTest`. Adding a `PARTITION BY` test should not require changes to the existing tests at all.

Reply:
Also I am not sure where the actual partition by test was actually added, un less I can't use CTRL+F in the browser.